mkdocs.yml

```diff
@@ -23,18 +26,25 @@ theme:
       scheme: slate
       toggle:
         icon: material/brightness-4
-        name: Switch to system preference
-site_description: |
-  Scarf's Technical Documentation & API Reference
+        name: Switch to light mode
+  font:
+    text: Roboto
+    code: Roboto Mono
+  favicon: assets/new_scarf.svg
```

Comment:
I think we may want to pick a different color for the favicon, at least for me it's either hard to see or fully invisible when the tab is currently selected:

![Screenshot 2023-07-31 at 1 46 30 PM](https://github.com/scarf-sh/docs/assets/1388071/b299d0b9-02da-4f03-8fbb-0803f20209ba)
![Screenshot 2023-07-31 at 1 46 24 PM](https://github.com/scarf-sh/docs/assets/1388071/067713a8-8612-42ee-ae65-777bdebb96ab)

Maybe we should do blue? Or black? Or non-transparent background? I don't feel strongly.

Reply:
Switched this to the blue icon
![icon](https://github.com/scarf-sh/docs/assets/4097509/74458eb9-ea62-4c29-ae6f-7f4677313bb0)